controllers/userController.js

```diff
@@ -1,36 +1,41 @@
 const User = require('../models/User');
 const { StatusCodes } = require('http-status-codes');
 const CustomError = require('../errors');
-const { createTokenUser, attachCookiesToResponse, checkPermissions } = require('../utils');
+const { createTokenUser, attachCookiesToResponse, checkPermissions } = require(
+  '../utils');
 
 // only admins should be able to see all users + we remove password from the response
+// AKOSREVIEW + we only return non-admin users
 const getAllUsers = async (req, res) => {
   console.log(req.user); //we pass to the browser object with name, userId ad role - check authentication.js line 13
-  const users = await User.find({ role: 'user' }).select('-password');
+  // AKOSREVIEW we can probaby also remove __v
+  const users = await User.find({ role: 'user' }).select('-password -__v');
```

Comment:
It probably isn't useful to anyone using this API to return the `_v` field since that's only used by Mongo internally.